src/hotspot/share/runtime/frame.cpp

```diff
@@ -301,6 +302,18 @@ bool frame::is_entry_frame_valid(JavaThread* thread) const {
   return (jfa->last_Java_sp() > sp());
 }
 
+Method* frame::safe_interpreter_frame_method() const {
+  Method** m_addr = interpreter_frame_method_addr();
+  if (m_addr == nullptr) {
+    return nullptr;
+  }
+  Method* m = (Method*) SafeFetchN((intptr_t*) m_addr, 0);
+  if (m == 0) {
```

Comment:
Small nits:
- nullptr instead of 0 
- you can just return m. Its either nullptr or the Method.

Reply:
This didn't compile: `error: cannot convert 'std::nullptr_t' to 'intptr_t' {aka 'long int'}` - I put it back to the literal 0.

Reply:
`(void*)nullptr` is probably best, with `0` it is not directly obvious, that it is a pointer.